lib/modules/apostrophe-any-page-manager/public/js/chooser.js

```diff
@@ -5,6 +5,13 @@ apos.define('apostrophe-any-page-manager-chooser', {
     var superDecorateManager = self.decorateManager;
     self.decorateManager = function(manager, options) {
       superDecorateManager(manager, options);
+      // Don't bomb if reorganize was overridden fully before this option was added
+      manager.options.jqtree = manager.options.jqtree || {};
+      // Don't cause "oops, reference to same object" problems for the next instantiation
+      manager.options.jqtree = _.clone(manager.options.jqtree);
```

Comment:
This could be done in the previous line, right? `= _.clone(manager.options.jqtree) || {}`